File: algorithm/34. Search for a Range/34.py
class Solution(object):
    def searchRange(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        result=[]
        start,end=-1,-1
        for i in xrange(len(nums)):
            if nums[i]<target:
                continue
            if nums[i]==target:
                if start==-1:
                    start=i
                end=i
            if nums[i]<target:
                break
        result.append(start)
        result.append(end)
        return result

File: algorithm/34. Search for a Range/34.cpp
class Solution {
public:
    vector<int> searchRange(vector<int>& nums, int target) {
        vector<int> result;
        int start=-1,end=-1;
        for (int i = 0; i< nums.size();i++)
        {
            if(nums[i]<target)
                continue;
            if(nums[i]==target)
            {
                if(start==-1)
                    start=i;
                end=i;
            }
            if(nums[i]>target)
                break;
        }
        result.push_back(start);
        result.push_back(end);
        return result;
    }
};
